project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -52,7 +52,7 @@ export default class HeatMap extends React.Component {
   }
 
   asyncInvokeDashboardRefresh(viewport) {
-    if (this.refs.map) {
+    if (this.refs.map && !this.state.mapMovedByProps) {
```

Comment:
`mapMovedByProps` is never set so it'll always be undefined which means that this conditional will always be true. So can we remove it?

Reply:
Removing it breaks the share link for some reason, can we leave it?

Reply:
In what way does it break the functionality? We should get to the bottom of this as it might indicate a design issue... this is the only occurrence of `mapMovedByProps` in this file so it _must_ be undefined. I'd like to get to the bottom of this magic.

Reply:
@c-w you're actually right, I just double checked again. I'll remove it.